### PR TITLE
to https, to adaptive, remove duplicates, clean-up unsorted.m3u

### PR DESCRIPTION
**Pull request by link8k**

### CODE REVIEW

File: channels/unsorted.m3u
Comment (link8k) [inline channels/unsorted.m3u:431]: This exists in https://github.com/iptv-org/iptv/blob/d1d51f0f2283836bc70ff7578d9344c4f4ef4e53/channels/co.m3u#L27 which is why it was removed.

I converted the links to adaptive and then ran "npm run format"

File: channels/af.m3u
Comment (themak1985) [inline channels/af.m3u:15]: New link does not work, original link works.

File: channels/al.m3u
Comment (themak1985) [inline channels/al.m3u:3]: New link does not work, original link works.

File: channels/ar.m3u
Comment (themak1985) [inline channels/ar.m3u:45]: New link does not work, original link works.

**Review by themak1985 — COMMENTED** [review]
Hi there @link8k, thanks for your contributions, I can tell its a lot of work. I made a few comments and decided to hold off on a full review until I get some clarity from you on some things.

- What is the rationale for moving to https? In my experience, that could cause issues for some people. 
- I also found that many of the links you transitioned to https, you also removed the proxy, which contributed to breaking the link. 
- Why did you change some of the chunk/chunklist links to playlist links? 
- How did you check all your links? I suggest using VLC and also sites like streamtest.in
- If a channel has a 480P and 1080P stream, it is okay to have both if they are labeled appropriately. It's helpful for those with unreliable internet.  The same applies for backups, but I don't believe you removed any of those. 

Thanks again for your contribution, looking forward to your reply.

File: channels/af.m3u
Comment (link8k) [reply channels/af.m3u:15]: Hi, this one works fine in VLC & via streamtest https://streamtest.in/logs/https-playout395-livestreamingcdn-com-live-stream1-playlist-m3u8-95d3da00-de0c-4e68-a3d7-e514cf53cd1c

File: channels/al.m3u
Comment (link8k) [reply channels/al.m3u:3]: Hi, this one also works fine in VLC & via streamtest https://streamtest.in/logs/http-176-31-74-164-8081-abcnewslive-abcnews-playlist-m3u8

File: channels/ar.m3u
Comment (link8k) [reply channels/ar.m3u:45]: This works for me via VLC

**Comment by link8k** [comment]
Hi @themak1985 , thanks for your comments and your previous contributions!

- What is the rationale for moving to https?

Security and a way of minimising duplicates
- I also found that many of the links you transitioned to https, you also removed the proxy, which contributed to breaking the link.

These work for me and via streamlink.in
- Why did you change some of the chunk/chunklist links to playlist links?

-- To convert them to an adaptive playlist, some streams have 10 bitrates and the player should pick based on the connection.
-- You can use "--adaptive-logic=highest" in VLC to force a specific quality
-- playlist.m3u8 is less likely to break, if the stream changes the qualities it offers / updates the streamer software then someone will have to fix the link. e.g. National Geographic Abu Dhabi changed the max quality from 1080p to 1024p. If "chunklist_b4000000_t64MTA4MHA=.m3u8" was specified then that would have broken.
-- Makes the list shorter
How did you check all your links? I suggest using VLC and also sites like streamtest.in

-- I wrote a python script which uses "streamlink"
- If a channel has a 480P and 1080P stream, it is okay to have both if they are labeled appropriately. It's helpful for those with unreliable internet. The same applies for backups, but I don't believe you removed any of those.

The problem is most aren't labelled and it's a guessing game. I don't want to watch a 1080p stream when I'm on mobile.
I think backups are fine as they're from a different source.

File: channels/ar.m3u
Comment (link8k) [reply channels/ar.m3u:45]: I'm happy to revert the https removal if that's what the maintainers want.

**Review by themak1985 — APPROVED** [review]

File: channels/us.m3u
Comment (themak1985) [inline channels/us.m3u]: Reverting to 1080P Stream

File: channels/us.m3u
Comment (themak1985) [inline channels/us.m3u:622]: Working Link

File: channels/us.m3u
Comment (themak1985) [inline channels/us.m3u:672]: Revert, Works

**Review by themak1985 — APPROVED** [review]
Made some changes, good to go now.

**Pull request merged**
